pkg/token/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package token defines tokens that can be used as identity in RBAC authorization.
```

Comment:
```suggestion
// Package token defines tokens that serve as identifiers in RBAC authorization.
```

Reply:
fixed in commit of next PR: https://github.com/WindzCUHK/k8s-athenz-sia/pull/11/commits/ce4b2bfc0d90baa210eff8a1f46cac652c4cf211